start-testnet-reth-miner-node.sh

```
@@ -0,0 +1,51 @@
+
+get_enode() {
+  for ((i = 1; i <= retries; i++)); do
+    response=$(curl -s -X POST --data '{"jsonrpc":"2.0","method":"admin_nodeInfo","params":[],"id":1}' \
+      -H "Content-Type: application/json" "http://$BOOTNODE_IP:$BOOTNODE_RPC_PORT")
+
+    echo "Response from http://$BOOTNODE_IP:$BOOTNODE_RPC_PORT"
+    echo "$response"
+
+    enode=$(echo "$response" | jq -r '.result.enode')
+
+    if [[ -n "$enode" ]]; then
```

Shellcheck (0.10.0)

[warning] In POSIX sh, [[ ]] is undefined.

(SC3010)